src/helpers/objectDiff.ts

```diff
@@ -0,0 +1,22 @@
+import { PlainObject } from "../types";
+import { isSame } from "../validators/isSame";
+import { getKeys } from "./getKeys";
+
+export const objectDiff = (
+  leftObject: PlainObject,
+  rightObject: PlainObject
+) => {
+  var diff: PlainObject = {};
+  const allKeys = new Set([...getKeys(leftObject), ...getKeys(rightObject)]);
+
+  for (const key of allKeys) {
+    if (!isSame(rightObject[key], leftObject[key])) {
+      diff[key] = {
+        from: leftObject[key],
+        to: rightObject[key],
```

Comment:
@mshd what do you think about this, it's like the function you made but instead of showing only the new value, it returns the old one as well, so in case there is a ui that wants to do something like ~~old~~ => new or the github diff